src/stores/AppStore.ts

```diff
@@ -170,27 +172,34 @@ export class AppStore {
   private splitGroup() {
     const groups = this.tabGroups;
     const lastGroup = groups[groups.length - 1];
-    if (lastGroup.files.length === 0) {
+    if (lastGroup.views.length === 0) {
       return;
     }
-    const group = new Group(lastGroup.file, null, [lastGroup.file]);
+    const group = new Group(lastGroup.view, [lastGroup.view]);
```

Comment:
do we clone group here? can we add the `clone()` method, or comment here?

Reply:
We are just taking the last file from the group, it's not a full clone, so `clone` would be misleading?

Reply:
Can we add "Take the last file from the group" a comment above?